core/schema/module.go

```diff
@@ -303,20 +320,23 @@ func (s *moduleSchema) typeDef(ctx context.Context, _ *core.Query, args struct{}
 
 func (s *moduleSchema) typeDefWithOptional(ctx context.Context, def *core.TypeDef, args struct {
 	Optional bool
-}) (*core.TypeDef, error) {
+},
+) (*core.TypeDef, error) {
```

Comment:
nit: this formatting shouldn't change - there's a few instances of this in this file.

Reply:
normally fixed